ReactiveArchitecture/Classes/nowplaying/service/Results.swift

```diff
@@ -25,37 +25,37 @@ import UIKit
 import ObjectMapper
 
 class Results: Mappable {
-    var poster_path: String?
-    var adult: Bool?
-    var overview: String?
-    var release_date: String?
-    var genre_ids: [Int]?
-    var id: Int?
-    var original_title: String?
-    var original_language: String?
-    var title: String?
-    var popularity: Double?
-    var vote_count: Int?
-    var video: Bool?
-    var vote_average: Double?
+    var posterPath: String!
+    var adult: Bool!
+    var overview: String!
+    var releaseDate: String!
+    var genreIds: [Int]!
+    var id: Int!
+    var originalTitle: String!
+    var originalLanguage: String!
+    var title: String!
+    var popularity: Double!
+    var voteCount: Int!
+    var video: Bool!
+    var voteAverage: Double!
```

Comment:
Force unwraps here rather than scattered throughout the code. The real solution to this is to convert to `Codable` but had no time

Reply:
I might try that on another PR. For now this is good for educational tool